File: src/main/java/io/sahil/f1dashboard/Model/Circuit.java
package io.sahil.f1dashboard.Model;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Circuit {

    @Id
    private String circuitId;
    private String circuitRef;
    private String name;
    private String location;
    private String country;
    private String lat;
    private String lng;

    
    public String getCircuitId() {
        return circuitId;
    }
    public void setCircuitId(String circuitId) {
        this.circuitId = circuitId;
    }
    public String getCircuitRef() {
        return circuitRef;
    }
    public void setCircuitRef(String circuitRef) {
        this.circuitRef = circuitRef;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public String getLocation() {
        return location;
    }
    public void setLocation(String location) {
        this.location = location;
    }
    public String getCountry() {
        return country;
    }
    public void setCountry(String country) {
        this.country = country;
    }
    public String getLat() {
        return lat;
    }
    public void setLat(String lat) {
        this.lat = lat;
    }
    public String getLng() {
        return lng;
    }
    public void setLng(String lng) {
        this.lng = lng;
    }

    


    
}

File: android/app/src/main/java/io/sahil/f1dashboard/data/models/Circuit.kt
package io.sahil.f1dashboard.data.models

import com.google.gson.annotations.SerializedName

data class Circuit(
    @SerializedName("circuitId")
    val circuitId: String? = "",
    @SerializedName("circuitRef")
    val circuitRef: String? = "",
    @SerializedName("name")
    val name: String? = "",
    @SerializedName("location")
    val location: String? = "",
    @SerializedName("country")
    val country: String? = "",
    @SerializedName("lat")
    val lat: String? = "",
    @SerializedName("lng")
    val lng: String? = ""
){
    override fun toString(): String {
        return "CircuitId: $circuitId, CircuitRef: $circuitRef, name: $name, location: $location" +
                ", country: $country, lat: $lat, lng: $lng"
    }
}